### PR TITLE
Assigmet 4

### PR DESCRIPTION
**Pull request by CesarPRinso**

### CODE REVIEW

File: Assignment4/CesarPRinso-223073965/Task06.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+# %% [markdown]
+# **TASK 6.4: Add to the individual JaneSmith the fullName, given and family names**
+
+# %%
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.JaneSmith, VCARD.FN, Literal('Jane Smith', datatype=XSD.string)))
+g.add((ns.JaneSmith, VCARD.Given, Literal('Jane', datatype=XSD.string)))
+g.add((ns.JaneSmith, VCARD.Family, Literal('Smith', datatype=XSD.string)))
+
```
Comment (anaigmo) [inline Assignment4/CesarPRinso-223073965/Task06.py:65]: missing email property also :)

File: Assignment4/CesarPRinso-223073965/Task07.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        'rdfs': RDFS
+    })
+
+# Visualize the results
+for r in g.query(q1):
+    print(r)
+
+# %%
+for subclass in g.subjects(RDFS.subClassOf, ns.Person):
+    print(subclass)
```
Comment (anaigmo) [inline Assignment4/CesarPRinso-223073965/Task07.py:42]: missing recursive subclass relationships

File: Assignment4/CesarPRinso-223073965/Task07.py
Comment (anaigmo) [inline Assignment4/CesarPRinso-223073965/Task07.py:1]: half the exercises in the task are missing or are not what is asked, check: https://github.com/FacultadInformatica-LinkedData/Curso2023-2024-ODKG/blob/master/Assignment4/course_materials/python/task07.py

**Comment by SerPablo** [comment]
Missing exercises, errors not solved

**Pull request closed without merging**